cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/server/CmisEventDispatcher.java

```diff
@@ -99,6 +111,29 @@ else if (XmlUtils.isWellFormed(result, "cmis")) {
 	}
 
 	public boolean contains(CmisEvent event) {
-		return eventListeners.containsKey(event);
+		if(StringUtils.isNotEmpty(dispatcherName)) {
+			JavaListener listener = JavaListener.getListener(dispatcherName);
+			if(listener == null) {
+				throw new CmisRuntimeException("unable to bridge cmis request, dispatcher offline"); //Adapter registered but not started
+			}
+
+			HashMap<String, Object> messageContext = new HashMap<>();
+			messageContext.put("CmisEvent", event.name());
+
+			try {
+				listener.processRequest(null, event.name(), messageContext);
+				// Nothing is done with the result, it's purely to check if the event should be bridged or not. If not, an execption should be thrown.
+
```

Comment:
Waarom geef je niet zoiets als true of false terug, en zodat je excepties voor excepties kan bewaren?

Reply:
Omdat je bij een exceptie echt wilt dat hij niets doet, en je mooi een foutmelding mee kan geven.

Reply:
always false unless explicitly true